internal/util/get/get.go

```diff
@@ -65,6 +74,37 @@ func (c Command) Run() error {
 	return nil
 }
 
+// updateParentKptfile searches the parent folders of a Kptfile. If it finds
+// a Kptfile, it means the package should be registered as a subpackage of the
+// parent. It adds the new package to the parent. The function returns a function
```

Comment:
We should update the entry in subpackages section of the parent package Kptfile if the child package is updated without the knowledge of the parent. But `get` command should not add the entry to the parent Kptfile.

Reply:
The main reason for adding this functionality is to simplify the different ways users can assemble composite packages. By always updating the parent Kptfile, we no longer have two ways of adding remote subpackages. This also essentially means that the cli is just porcelain on top of the declarative model.

Reply:
Agree. I am not able to see the shortcomings of this approach for now but wondering why other tools like Helm support 2 ways of adding subchats.